dynamic.js

```diff
@@ -279,6 +279,9 @@ function genResponse (response) {
 
   Object.keys(response).forEach(key => {
     if (response[key].type) {
```

Comment:
I am not really familiar with swagger but just peeking at examples I don't think `type` is mandatory. If `type` is not required, maybe we can remove `if (response[key].type)` all together to solve the problem?

Any thoughts?

Reply:
The problem when remove the type in schema like bellow 
```
schema: {
      response: {
        204: {
          description: 'Succesful response',
        },
      },
    },
```
It shows an error on fastify validation 'fast-json-stringify'

Any thoughts?